### DISCUSSION TITLE
How to export the arXiv archiveprefix field with a capital X?

### DISCUSSION THREAD

**Discussion by alves-nickolas** [retorquere/zotero-better-bibtex#2905] [Q&A] [answered]
In many of the papers I add to Zotero, I add the arXiv link by adding a line similar to `arXiv: 2305.17453 [gr-qc]` to the "extra" Zotero field. This appropriately leads to the correct arXiv information in the .bib file both in BibLaTeX and in BibTeX. 

There is, however, something that often bothers me: in BibTeX, the typical entry will get the field `archiveprefix = {arxiv}`. When I compile my LaTeX files, this very often leads to the bibliography entries reading as "arxiv: 2305.17453 [gr-qc]" with a lowercase X, but I would much rather have an uppercase X, which is the usual style for arXiv. After testing with the LaTeX code, I noticed that changing the .bib fyle to have `archiveprefix = {arXiv}` (with an uppercase X) will solve my problem just fine. 

For one or two entries, I could surely do this manually. But of course, when you have hundreds of references with arXiv links I would rather have an automated system. 

How can I make Better BibTeX export the information to the BibTeX .bib file with the uppercase X in "arXiv" in the `archiveprefix` field?

**Comment by retorquere** [comment, marked as answer]
this is reasonable, I've made `arXiv` the default in the new release.

**Reply by alves-nickolas** [reply]
I'm not very used to GitHub. How can I download this update? Will it be available in a future release or is it available already?

**Reply by retorquere** [reply]
It will go out tonight. You can select "check for updates" in the addons menu, or wait for zotero to update automatically (within 48 hours of release)
